app/src/main/java/com/wizeline/academy/animations/core/data/repository/HomeRepository.kt

```diff
@@ -32,13 +34,18 @@ class HomeRepository @Inject constructor() {
         1025
     )
 
-    fun fetchMockedImages(): List<Int> {
-        val images = mutableListOf<Int>()
+    fun fetchMockedImages(): List<HomeItem> {
+        val items = mutableListOf<HomeItem>()
         for (i in 1..10) {
-            val randomId = getRandomId()
-            images += randomId
+            val item = createNewHomeItem()
```

Comment:
you could try something like
val items = (1..10).map { createNewHomeItem() }

Reply:
Thanks I already fixed it, it is way better.